srcpkgs/mstflint/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'mstflint'
+pkgname=mstflint
+version=4.11.0.3
+revision=1
+wrksrc="${pkgname}-4.11.0"
+build_style=gnu-configure
+configure_args="--enable-fw-mgr --enable-openssl"
+makedepends="boost-devel libcurl-devel libibmad-devel liblzma-devel \
+	libressl-devel libxml2-devel rdma-core-devel zlib-devel"
```

Comment:
please use 1 whitespace prefix on newlines being derived from a line being cut for too long